database/person/queries/search_people.sql

```diff
@@ -0,0 +1,18 @@
+SELECT *
+FROM (
+		SELECT people.id,
+			concat(people.name, ' ', people.surname) AS name,
+			string_agg(emails.email, ', ') AS emails,
+			string_agg(categories.name, ', ') AS categories,
+			GREATEST(
+				similarity(people.name, %(name)s),
+				similarity(people.name, %(name)s)
```

Comment:
These two lines are the same? How does it work?

Reply:
Oops it's meant to be surname